docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+🔲 Walk away with a introductory understanding of the *CIT* developer workflow.
+
+🔲 Feel confident in moving forward to our next recommended section: *[Cobalt Templating from Scratch](https://github.com/microsoft/cobalt/blob/master/docs/3_NEW_TEMPLATE.md).*
 
 ## 2.3 Prerequisites
-< Enumerate any prerequisite walkthroughs that should be completed prior to this, and any technical prerequisites such as developer environment / tools >
 
-## 2.4 Walkthrough
-< Step-by-step instructions for completing this walkthrough >
\ No newline at end of file
+> NOTE: Previous "infrastructure as code" experience is not a prerequisite for completing the quick start guide.
+
+| Azure Prereqs | Description | I | Install Prereqs | Description |
+|----------|--------------|-|----------|--------------|
+| `Azure Subscription` | [Azure Portal](https://portal.azure.com/) - This template needs to deploy infrastructure within an Azure subscription.|I|`Terminal with bash shell`|[WSL](https://code.visualstudio.com/docs/remote/wsl) or [Git Bash](https://git-scm.com/downloads) - The shell environment needed to follow along with the provided instructions.|
+|`Azure Service Principal`|[Azure Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) - This template needs permissions to deploy infrastructure within an Azure subscription.|I|`Git Source Control`|[Install git](https://www.atlassian.com/git/tutorials/install-git)|
+|`Azure Storage Account`|[Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) - An account for tracking terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.|I|`Terraform`|[Terraform](https://www.terraform.io/downloads.html) - Download the appropriate environment for setting up Terraform|
+|`Azure CLI`|[Get started with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest)- An account for tracking terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.|I|`GitHub Account`|[Github](https://github.com/login) - An account for forking the Cobalt repo|
+
+---
+
+## 2.4 Deploy Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+Below are the steps for deploying the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md). Ensure that this deployment outputs the following Azure cloud resources before you can call this quick start guide finished:
+
+| Preview | Quick Start Azure Cloud Resources |
+|----------|--------------|
+|![cobalt-'NickeManarin/Screen2Gif'](./screentogif-40sec-test.gif)| <image src="https://user-images.githubusercontent.com/10041279/66533295-0fc64b80-ead8-11e9-96c0-ea5441cde854.png" width="1400"/> |
```

Comment:
Gif seems out of place here -- does it make sense to have each step have its own step-specific gif so that it is more clear what is happening?

Reply:
A gif for each step would be overwhelming in my opinion. I think what matters is how the header above the gif is title in order to convey the correct context. I'll think about this a bit more. Thanks.